pubspec.yaml

```diff
@@ -14,10 +14,7 @@ dependencies:
   app_installer: ^1.1.0
   collection: ^1.17.0
   cross_connectivity: ^3.0.5
-  cr_file_saver:
-    git:
-      url: https://github.com/dhruvanbhalara/cr_file_saver.git
-      ref: a08326ecb48f581b4b09e2e2665d31ed1704c7af
```

Comment:
```yaml
  cr_file_saver:
    git:
      url: https://github.com/dhruvanbhalara/cr_file_saver
      ref: "fix/incorrect_file_name"
```
@andrisas please try with the latest code it's solved now.

Reply:
i tried copying your code and changing ref to 410703245af7ca37421ccd7e8e37281a4e57026f
the same issue ocures in both cases

Reply:
> i tried copying your code and changing ref to 410703245af7ca37421ccd7e8e37281a4e57026f
> the same issue ocures in both cases

Can you please show block of code of `cr_file_saver` in `pubspec.lock`?

Reply:
> Can you please show block of code of `cr_file_saver` in `pubspec.lock`?

 cr_file_saver:
    dependency: "direct main"
    description:
      path: "."
      ref: "410703245af7ca37421ccd7e8e37281a4e57026f"
      resolved-ref: "410703245af7ca37421ccd7e8e37281a4e57026f"
      url: "https://github.com/dhruvanbhalara/cr_file_saver"
    source: git
    version: "0.0.2+1"

Reply:
If possible can you get device side logs from Android studio Logcat and share it with me in discord server?

Reply:
Deleting /home/USERNAME/.pub-cache fixed it